awesomplete.js

```diff
@@ -39,10 +39,12 @@ var _ = function (input, o) {
 
 	// Create necessary elements
 
-	this.container = $.create("div", {
-		className: "awesomplete",
-		around: input
-	});
+	this.container = input.parentNode.className.search(/awesomplete-wrapper/) >= 0 ?
```

Comment:
why not `classList.contains("awesomplete-wrapper")`?
(if the answer is "browser support", IE9 can go fu…erm use a polyfill)